integration-tests/tests/src/tests/sync/sync-as-local.ts

```diff
@@ -33,7 +33,6 @@ describe.skipIf(environment.missingServer, "Synced Realm as local", () => {
   });
 
   before(async function (this: RealmContext) {
-    this.timeout(5000);
```

Comment:
We should do something else here, to make it stable when running locally then?

Reply:
I have tried to find a way to set a default, but haven't been successful.  Perhaps we should just set it in the `package.json`

Reply:
Let's look this over later